Control coverage/Feature/SubscriptionCore.md

```diff
@@ -1,14 +1,933 @@
+
+### Control Spec 
+
+> **Passed:** 
+> ASC assessment status is healthy.
+> 
+> **Failed:** 
+> ASC assessment status is unhealthy. (or) ASC assessment status is "NotApplicable" with "cause" as either "OffByPolicy" or "Exempt".
+> 
+> **Verify:** 
+> ASC assessment status is not applicable (with "cause" other than "OffByPolicy" and "Exempt"), OR ASC assessment status was not found.
+> 
+
+### Recommendation 
+
+- **Azure Portal** 
+
+	 You need to remove any management certificates that are not required. Please follow these steps: <br />(a) Logon to https://portal.azure.com/ <br />(b) Navigate to Subscriptions <br />(c) Select the subscription <br />(d) Go to Settings tab  --> Management Certificates tab --> Delete unwanted management certificates. 
+
+### Azure Policy or ARM API used for evaluation 
+
+- ARM API to list security assessments at subscription level: - /subscriptions/{subscriptionId}/providers/Microsoft.Security/assessments?api-version=2020-01-01 <br />
+**Properties:** id, name, resourceDetails.Id, displayName, status.code, status, additionalData
+ <br />
+
+<br />
+
+___ 
+
+## Azure_Subscription_Audit_Resolve_Azure_Security_Center_Alerts 
+
+### DisplayName 
+Pending Azure Security Center (ASC) alerts must be resolved 
+
+### Rationale 
+Based on the policies that are enabled in the subscription, Azure Security Center raises alerts (which are typically indicative of resources that ASC suspects might be under attack or needing immediate attention). It is important that these alerts/actions are resolved promptly in order to eliminate the exposure to attacks. 
+
+### Control Settings 
+```json 
+{
+    "ASCAlertsGraceInDays": {
+        "High": 0,
+        "Medium": 30
+    }
+}
+ ```  
+
+### Control Spec 
+
+> **Passed:** 
+> There are no active ASC Alerts OR there is no active alert which is beyond defined grace.
+> 
+> **Failed:** 
+> There are ASC alerts in the subscription which are active beyond the defined grace. <br />Alert Severity: High, Grace period: 0 <br />Alert Severity: Medium, Grace period: 30
+> 
+
+### Recommendation 
+
+- **Azure Portal** 
+
+	 You need to address all active alerts on Azure Security Center. Please follow these steps: <br />(a) Logon to https://portal.azure.com/ <br />(b) Navigate to Security Center. <br />(c) Click on Security Alerts under 'Threat Protection' category. <br />(d) Take appropriate actions on all active alerts. 
+
+### Azure Policy or ARM API used for evaluation 
+
+- ARM API to list all the alerts that are associated with the subscription: - /subscriptions/{subscriptionId}/providers/microsoft.Security/alerts?api-version=2015-06-01-preview <br />
+**Properties:** properties.state, properties.reportedSeverity, properties.reportedTimeUtc
```

Comment:
Should this be `[*].properties...`, considering the response is a list of alerts. `[*]` to indicate - for all alerts.